### PR TITLE
added get endpoint for articles and article_id with tests

### PR DESCRIPTION
**Pull request by s-hackman**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -45,8 +45,108 @@ describe("GET /api/topics", () => {
...
+describe("GET /api/blah", () => {
+  it("responds with status 400 for bad path message", () => {
+    return request(app)
+      .get("/api/blah")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("URL not found");
+      });
+  });
+});
```
Comment (NRMSMITH) [inline __tests__/app.test.js:63]: Is this supposed to be 400 or 404?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -45,8 +45,108 @@ describe("GET /api/topics", () => {
...
+      });
+  });
+  it("responds with status 400 for invalid non numeric id", () => {
+    return request(app)
+      .get("/api/articles/banana")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("invalid id");
+      });
+  });
```
Comment (NRMSMITH) [inline __tests__/app.test.js:151]: 👍

**Review by NRMSMITH — APPROVED** [review]
Slight tweak on a test - but you've got the all clear from me!

File: __tests__/app.test.js (hunk 1 above)
Comment (s-hackman) [reply __tests__/app.test.js:63]: I believe this is 404 as it is wrong path for api call

**Pull request merged**